docs/source/querying_data/remote_processors.md

```diff
@@ -0,0 +1,77 @@
+# Remote Search Processors
+
+> 👉 This page assumes you’re familiar with OpenSearch’s concept of a search pipeline and search processor
+
+Sycamore exposes a number of search processors in addition to the ones built in to OpenSearch. We accomplish this with a search processor in OpenSearch called ‘remote-processor’, which makes network calls to a service hosting the sycamore search processors.
+
+These search processors include:
+
+- `debug`: prints the search response to stdout. Useful for debugging.
+- `dedup`: works in conjunction with the `Sketcher` ingest transform to deduplicate search results at query-time. See the [sketch](../data_ingestion_and_preparation/transforms/sketch.md) transform for more details.
```

Comment:
can bump it. was going with alphabetical scheme but can go by importance scheme. I couldn't find anywhere to point to other than sketch. alex confirmed this

Reply:
Yes - it's because Alex hasn't written it yet. That's where all the info is. Can you please add this in today once Alex's docs for query time NDD are written? We want users going to that page, not sketcher.

Reply:
Alphabetical is fine, I guess.

Reply:
will re-point with a new target. Actually will make alex re-point this.